demos/c2-test/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "supraseal-c2-test"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+
+[dev-dependencies]
+supraseal-c2 = "0"
+anyhow = "1.0.26"
+bincode = "1.1.2"
+filecoin-proofs-api = "15.0.0"
+filecoin-proofs-v1 = { package = "filecoin-proofs", version = "15.0.0", default-features = false }
+storage-proofs-core = { version = "15.0.0", default-features = false }
```

Comment:
Just a note: ideally you depend only on `filecoin-proofs-api` and not the rest. This is our public API, the rest is kind of for internal use.

Though that's not a blocker and I'm happy to provide a PR myself cleaning things up, once we have a release of `supraseal-c2`.

Reply:
Thanks! (And I've updated the README.md:-)